hmpps-sqs-spring-boot-autoconfigure/build.gradle.kts

```diff
@@ -2,19 +2,22 @@ import com.github.benmanes.gradle.versions.updates.DependencyUpdatesTask
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
-  kotlin("jvm") version "1.9.22"
-  kotlin("plugin.spring") version "1.9.22"
+  kotlin("jvm") version "1.9.23"
+  kotlin("plugin.spring") version "1.9.23"
   id("maven-publish")
   id("signing")
   id("com.adarshr.test-logger") version "4.0.0"
   id("com.github.ben-manes.versions") version "0.51.0"
   id("se.patrikerdes.use-latest-versions") version "0.2.18"
   id("io.spring.dependency-management") version "1.1.4"
   id("org.jlleitschuh.gradle.ktlint") version "12.1.0"
-  id("org.owasp.dependencycheck") version "8.4.3"
-  id("org.springframework.boot") version "3.2.2"
+  id("org.owasp.dependencycheck") version "8.4.2"
+  id("org.springframework.boot") version "3.2.4"
```

Comment:
assuming downgrade 8.4.3 -> 8.4.2 is intended

Reply:
oops no it isn't, accidentally did the same in the other lib but don't think it will make any difference